src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -30425,6 +30425,82 @@ void Main(object? o)
             comp2.VerifyDiagnostics(expected);
         }
 
+        [Fact, WorkItem(45795, "https://github.com/dotnet/roslyn/issues/45795")]
+        public void DoesNotReturn_ReturnStatementInLocalFunction()
+        {
+            var source = @"
+using System.Diagnostics.CodeAnalysis;
+#nullable enable
+public class C {
+    [DoesNotReturn]
+    public void M() {
+        _ = local1();
+        local2();
+        throw null!;
+        
+        int local1() { return 1; }
+        void local2() { return; }
+    }
+}
+";
+            var comp = CreateCompilation(new[] { source, DoesNotReturnAttributeDefinition });
+            comp.VerifyDiagnostics();
+        }
+
+        [Theory]
+        [InlineData("")]
+        [InlineData("static ")]
```

Comment:
Does `[DoesNotReturn]` analysis depend on whether the member is `static`?

Reply:
No, I pushed this by accident, I believe this function should be deleted.

Reply:
Simplified the test down to a `[Fact]` to simplify, since 'static' differences are not expected to impact the test.